.github/workflows/rust.yml

```diff
@@ -0,0 +1,33 @@
+name: Rust
+
+on: [push]
```

Comment:
Wouldn't it be better to only run in pushes to the `master` branch?

Reply:
Actually it's useful for any branch I think. For situations when building might fail on a platform you don't have access to

Reply:
Yeah, makes sense 👍